mitodl/ocw-hugo-themes
# move resource shortcode (and the shortcodes it calls) to the base theme

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/691

#### What's this PR do?
In https://github.com/mitodl/ocw-hugo-projects/pull/161, we added the ability to embed resources in sites using the `ocw-www` starter, which uses the `www` theme.  This PR moves the `resource` shortcode templates to `base-theme` so they can be accessed by both themes.

#### How should this be manually tested?
 - Clone https://github.mit.edu/mitocwcontent/ocw-www on the `main` branch as well as any other course repo from `mitocwcontent`
 - In your `.env`, set the following:
```
WWW_CONTENT_PATH=/path/to/mitocwcontent/ocw-www/
COURSE_CONTENT_PATH=/home/gumaerc/Code/mitocwcontent/
OCW_TEST_COURSE=<your test course>
RESOURCE_BASE_URL=https://ocw.mit.edu/
STATIC_API_BASE_URL=https://ocw.mit.edu/
OCW_STUDIO_BASE_URL=https://ocw-studio.odl.mit.edu/
```
 - Run `npm run start:www`
 - Visit http://localhost:3000/pages/gene-stark/ and ensure you see the embedded image
 - Stop the dev server and run `npm run start:course`
 - Make sure your course builds and is accessible at http://localhost:3000
